store/storetest/webhook_store.go

```diff
@@ -168,6 +170,35 @@ func testWebhookStoreGetIncomingByTeam(t *testing.T, ss store.Store) {
 	}
 }
 
+func TestWebhookStoreGetIncomingByTeamFilterByUser(t *testing.T, ss store.Store) {
+	var appErr *model.AppError
+
+	o1 := buildIncomingWebhook()
+	o1, appErr = ss.Webhook().SaveIncoming(o1)
+	require.Nil(t, appErr)
+
+	o2 := buildIncomingWebhook()
+	o2.TeamId = o1.TeamId //Set both to the same team
+	o2, appErr = ss.Webhook().SaveIncoming(o2)
+	require.Nil(t, appErr)
+
+	// check both are returned first
```

Comment:
Can we separate this tests using individual `t.Run` calls?

Reply:
✅